### DISCUSSION TITLE
Plugins VS Code

### DISCUSSION THREAD

**Discussion by nelsonvilte** [Launch-X-Latam/MisionFrontEnd#10] [Q&A] [answered]
¿Que plugins recomiendan para VS Code?

**Comment by ManueLuna-Mx** [comment]
Prueba esta.
Flatland Monokai Theme y esta Live Server

**Comment by magouwuxd** [comment]
Error lens, prettier y snippets. Tal vez ESLint si sabes usarlo

**Reply by rfponce** [reply]
Error lens es muy útil. Te muestra los errores más claramente.

**Comment by RicardoMartinez96** [comment]
Bookmarks, Auto Rename Tag y Todo Tree

**Comment by AlvaroFierro** [comment, marked as answer]
Yo uso
- Auto close tag
- Auto rename tag
- Babel ES6/ES7
- Better Comments
- Code spell checker
- Error
- ESLint
- Git Graph
- Git Lens
- Highlight Matching tag
- Live server
- Markdown all in one
- npm
- Prettier
- Quokka.js{
- Wakatime

Además de los temas, yo actualmente tengo instalado United Ubuntu

**Reply by hoyola** [reply]
Hola, la extensión de Quokka en su versión Free o la de Pago?
Tengo la free he tratado de utilizarla pero no funciona, me podrias ayudar

**Reply by itzelpz** [reply]
Para poder activarlo debes abrir vs code. Presionas Ctrl + Shift + P.  Escribes Quokka.js: Star

**Comment by rfponce** [comment]
Además de los ya mencionados a mí me gusta mucho Material Icon Theme. Cambia los iconos de los archivos y a las carpetas también les pone un icono especial dependiendo de cómo la llames.

**Comment by doguedogue** [comment]
Mis favoritos Activitus Bar y VS Code Pets

![image](https://user-images.githubusercontent.com/23409026/192431015-149078bb-add7-4988-a35e-09a99252ffaa.png)

**Comment by NiiSV811** [comment]
Si te gusta el anime y/o las monas chinas, te recomiendo Doki Theme.
Para HTML, HTML snippets
Y existe un plugin y una opción del VS code para hacer las llaves, los parentesis y los corchetes de coleres
